frame/elections-phragmen/src/lib.rs

```diff
@@ -974,6 +974,8 @@ impl<T: Trait> Module<T> {
 
 			// compute the outgoing of runners up as well and append them to the `to_burn_bond`
 			{
+				new_runners_up_ids.sort();
+				old_runners_up_ids.sort();
```

Comment:
Is the computational effect great enough that we need to benchmark?

Reply:
Don't think so, these are both bounded to fairly small number e.g. 7, 13, 17 and such.